config/src/config/test_data/validator_full_node.yaml

```diff
@@ -33,6 +33,7 @@ full_node_networks:
           addresses:
             - "/ip4/127.0.0.1/tcp/58259/ln-noise-ik/c998dcd54c3daf48e0ad516d94b7be0b0b7a27caa00541f2b2c14b13500df10b/ln-handshake/0"
           keys: ["c998dcd54c3daf48e0ad516d94b7be0b0b7a27caa00541f2b2c14b13500df10b"]
+          role: "Validator"
```

Comment:
random comment: i wish there was a better way to document these yaml configs. If I'm a node operator just looking at the yaml itself, I have no idea what these fields do, no idea what fields are even possible, and no idea in what contexts they're useful. Even knowing that the NetworkConfig exists, it's difficult mapping rust structs <-> yaml in your head. I wonder if there's some way to generate these example configs including the doc comments? anyway, I'll stop rambling... : p

Reply:
We could add comments to YAML, that's kinda the point of using yaml :P

But, I know it's very confusing to figure this all out.